formgroups/contactpersons.html

```diff
@@ -10,7 +10,7 @@
           <div class="input-group has-validation">
             <div class="form-floating">
               <input type="text" class="form-control" id="input-contactperson-lastname" pattern="^[a-zA-ZäöüÄÖÜß \-]*$"
-                name="cpLastname[]" />
+                name="cpLastname[]"  onchange="checkMandatoryFields()"/>
```

Comment:
Hier hast du den Funktionsaufruf direkt in der HTML-Struktur implementiert. Das funktioniert natürlich so und würden wir nur JavaScript ohne jQuery nutzen, wäre das auch gut so. Aber mit jQuery haben wir ja eigentlich Event Handler zur Verfügung, die wir auch seit Beginn des Projekts nutzen. Weshalb hast du dich hier gegen einen Event Handler entschieden?
Die Vorteile gegenüber einer Implementierung innerhalb der HTML-Struktur sind halt:
- **Bessere Trennung von HTML und JavaScript:** HTML bleibt sauber und JavaScript wird an einem Ort verwaltet.
- **Bessere Wartbarkeit:** Änderungen an Event-Handlern können zentral im JavaScript-Code vorgenommen werden, ohne die HTML-Struktur zu ändern.
- **Wiederverwendbarkeit:** Event-Handler können leicht auf mehrere Elemente angewendet werden.
- Außerdem fände ich es gut, wenn wir da einheitlich bleiben könnten und wir haben halt von Anfang an Event Handler genutzt um HTML und JS konsequent zu trennen...